www/pages/privacy-policy/privacy-policy.html

```diff
@@ -0,0 +1,14 @@
+<ion-view view-title="Privacy Policy">
+  <ion-content>
+    <div class="list card item-text-wrap">
+      <div class="item item-text-center">
+        This application uses your geolocation as well as Google Analytics.
+        The data collected is the property of UMass Transit Services. Google Analytics data is solely used by the developers of this application for the purpose of continuing
```

Comment:
"...for the purpose of continuing improvements to..." doesn't sound like proper English?